src/appMain/CMakeLists.txt

```diff
@@ -155,7 +155,11 @@ if( NOT CMAKE_BUILD_TYPE )
 endif()
 
 add_executable(${PROJECT} ${HEADERS} ${SOURCES})
-target_link_libraries(${PROJECT} ${LIBRARIES})
+if(${QT_PORT})
+        target_link_libraries(${PROJECT} ${LIBRARIES} ${QT5Core})
+  else()
+        target_link_libraries(${PROJECT} ${LIBRARIES})
```

Comment:
May be add QT5Core lib to LIBRARIES list in case of QT_PORT and not use target_link twice?